From: Dmitry Smirnov <onlyjob@member.fsf.org>
Date: Tue, 26 Mar 2013 22:00:32 +0100
Subject: GG_unbundle_libltdl

Removes bundled "libltdl-dev" in order to use system one.

Forwarded: not-needed

---
 FreeOrion/GG/CMakeLists.txt       |   33 ---------------------------------
 FreeOrion/GG/GG/PluginInterface.h |    6 +-----
 FreeOrion/GG/src/CMakeLists.txt   |    9 ---------
 3 files changed, 1 insertion(+), 47 deletions(-)

diff --git a/FreeOrion/GG/CMakeLists.txt b/FreeOrion/GG/CMakeLists.txt
index e35ecec..7cb4599 100644
--- a/FreeOrion/GG/CMakeLists.txt
+++ b/FreeOrion/GG/CMakeLists.txt
@@ -190,41 +190,6 @@
     ${FREETYPE_INCLUDE_DIRS}
 )

-########################################
-# Header Generation                    #
-########################################
-if (BUILD_PLUGIN_SUPPORT)
-    if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/libltdl/config.h)
-        if (UNIX)
-            message("-- Configuring libltdl using \"libltdl/configure\" ...")
-            execute_process(
-                 COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/libltdl/configure
-                 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/libltdl
-                 OUTPUT_QUIET
-            )
-        else ()
-            file(WRITE ${CMAKE_CURRENT_SOURCE_DIR}/libltdl/config.h
-                 "/* WARNING: Generated by GG's build system.  All local changes will be lost! */
-#define error_t int
-#define HAVE_STDIO_H 1
-#define HAVE_STDLIB_H 1
-#define HAVE_STRING_H 1
-#define HAVE_CTYPE_H 1
-#define HAVE_MEMORY_H 1
-#define HAVE_ERRNO_H 1
-#define __WIN32__
-#define HAVE_MEMCPY 1
-#define HAVE_MEMMOVE 1
-#define LTDL_OBJDIR \".libs\"
-#define LTDL_DLOPEN_DEPLIBS 1
-#define LTDL_SHLIBPATH_VAR \"PATH\"
-#define LTDL_SHLIB_EXT \".dll\"
-"
-            )
-        endif ()
-    endif ()
-endif ()
-
 configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Config.h.in
     ${CMAKE_CURRENT_BINARY_DIR}/GG/Config.h
